release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -443,6 +443,29 @@ In {product-title} {product-version}, the installation program uses the Cluster
 ==== Cluster API replaces Terraform on {azure-first} Stack Hub installations
 
 In {product-title} {product-version}, the installation program uses the Cluster API instead of Terraform to provision clusters during installer-provisioned infrastructure installations on {azure-first} Stack Hub.
+[id="ocp-4-19-installation-and-update-support-azure-instance-types_{context}"]
+==== Support added for additional {azure-first} instance types
+
+Additional {azure-first} instance types for machine types based on 64-bit x86 architecture have been tested with {product-title} {product-version}.
+
+For the Dxv6 machine series, the following instance types have been tested:
+
+* `StandardDdsv6Family`
+* `StandardDldsv6Family`
+* `StandardDlsv6Family`
+* `StandardDsv6Family`
+
+For the Lsv4 and Lasv4 machine series, the following instance types have been tested:
+
+* `standardLasv4Family`
+* `standardLsv4Family`
```

Comment:
From the [comments](https://issues.redhat.com/browse/OCPSTRAT-1729?focusedId=27332993&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-27332993), seems that those series are GA after OCP GA.

Maybe need to state here they are in preview from Microsoft as Mak [commented](https://issues.redhat.com/secure/ViewProfile.jspa?name=mak.redhat.com)?

@makentenza WDYT?

Reply:
@brendan-daly-red-hat based on [comments](https://issues.redhat.com/browse/OCPSTRAT-1729?focusedId=27385974&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-27385974), those machine series have been released on Microsoft.